assets/sass/imports/_header.scss

```
@@ -251,3 +251,85 @@
     top: $container-padding;
   }
 }
+
+
+.menu-header {
+    display: flex;
+    flex-direction: row;
+}
+
+.menu-header .menu-item-header {
+    display: flex;
+    align-items: center;
+    font-size: 16px;
+    height: 45px;
+    background-color: inherit;
+    width: max-content;
+}
+
+.menu-header .dropdown  {
+    display: block;
+    font-size: 16px;
+    padding: 11px;
+    position: absolute;
+    top: 60px;
+    min-width: 160px;
+}
+
+.menu-header .dropdown .dropdown-item {
+    display: none;
+    position: relative;
+    padding: 5px 10px;
+    right: 10px;
+    vertical-align: middle;
+    align-items: center;
+    box-shadow: 2px 2px 2px rgba(0, 0, 0, 0.3);
```

```suggestion
    box-shadow: 4px 4px 4px rgba(0, 0, 0, 0.4);
```
It looks better with more blur